aea/crypto/cosmos.py

```diff
@@ -78,6 +78,38 @@ def is_transaction_settled(tx_receipt: JSONLike) -> bool:
                 )
         return is_successful
 
+    @staticmethod
+    def get_code_id(tx_receipt: JSONLike) -> Optional[int]:
```

Comment:
Also we will need to instantiate the ledgerApis obj in the skill in order to access the get_code_id. Since we have to pass the code_id in the message. 
```
                    contract_api_msg, contract_api_dialogue = contract_api_dialogues.create(
                        counterparty=LEDGER_API_ADDRESS,
                        performative=ContractApiMessage.Performative.GET_DEPLOY_TRANSACTION,
                        ledger_id=self.context.default_ledger_id,
                        contract_id="ddn/escrow:0.2.0",
                        callable="get_deploy_transaction",
                        kwargs=ContractApiMessage.Kwargs(
                            {
                                "label": "EscrowContract",
                                "init_msg": {},
                                "gas": 8000000,
                                "amount": 0,
                                "code_id": code_id  # from receipt,
                                "deployer_address": self.context.agent_address,
                            }
                        ),
                    )
```

Reply:
all you do is `code_id = Helper.get_code_id(tx_receipt)`. This is a static method.